unstructured/ingest/evaluate.py

```diff
@@ -164,6 +240,8 @@ def _display(rows, headers):
 
 
 def _mean(scores: List[float], rounding: Optional[int] = 3):
+    if len(scores) < 1:
+        return None
```

Comment:
are we returning None here because we want the result to be empty rather than 0 or something?

Reply:
Yes, because if there's no value, we do not want to say the average is zero. But let me also add another condition for 0 real quick.